open-metadata-implementation/governance-servers/open-lineage-services/open-lineage-services-api/src/main/java/org/odpi/openmetadata/governanceservers/openlineage/model/LineageEdge.java

```diff
@@ -27,16 +27,21 @@ public class LineageEdge implements Serializable {
 
     public LineageEdge(){}
 
+    protected Object id;
```

Comment:
why is this an Object - is it not always a String?

Reply:
Consistency is the reason for having id as an Object. Check LineageVertex